src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -628,7 +628,7 @@ std::string
 NetworkOPsImp::getHostId (bool forAdmin)
 {
     if (forAdmin)
-        return beast::getComputerName ();
+        return boost::asio::ip::host_name();
```

Comment:
Micro-optimization, but should we do:

    static std::string const hostname = boost::asio::ip::host_name();
    
    if (forAdmin)
        return hostname;